kommitters/calendso
# Send the email depending on the person who canceled the reservation.

## What does this PR do?



Fixes #1708
_The event shows up in both user's upcoming bookings_: the solution for this first bug is to edit the filter when querying the bookings in apps/Web/server/routers/viewer.tsx, even though the booking is not showing in B's bookings, it blocks the availability of B.
 Although, in my opinion, it is better if you could reschedule or cancel the reservation, even if you are not the organizer.
<img width="280" alt="image" src="https://user-images.githubusercontent.com/19562383/162875807-81327387-c8b3-4488-b73e-cfc4c947535d.png">


## Type of change




## How should this be tested?



For the first issue

1. Create 2 accounts, account A and B on cal.com.
2. Create an event in account A.
3. Schedule an event on account A using the email from account B.
4. Log in to account A and go to bookings/upcoming, you are going to see the booking.
5. Log in to account B and go to bookings/upcoming, you are not going to see the booking.

For the second issue

1. If an attendee cancels the reservation, the email will be sent to the organizer.
